frameworks/kafka/tests/test_sanity.py

```diff
@@ -349,9 +363,4 @@ def test_suppress():
     suppressed_url = urllib.parse.urljoin(
         dcos_url, 'service/{}/v1/state/properties/suppressed'.format(FOLDERED_SERVICE_NAME))
 
-    def fun():
-        response = dcos.http.get(suppressed_url)
-        response.raise_for_status()
-        return response.text == "true"
-
-    shakedown.wait_for(fun)
+    wait_for_response(suppressed_url)
```

Comment:
An out of scope comment. The suppressed url does not exist in 1.8.

Reply:
Strictly speaking `suppressed` will exist if the build available on 1.8 is new enough. It's served by the scheduler.

Reply:
My bad. It was my understanding that this was the reason that #1398 was needed (including skipping the suppressed check in 1.8 -- where I checked and this call would return 404).